.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true  # default
+contact_links:
+- name: '💬 IRC: #pypa @ Freenode'
+  url: https://webchat.freenode.net/#pypa
+  about: Chat with devs.
+- name: 🤷💻🤦 Discourse
```

Comment:
I would put Discourse first, since:

- it seems more active thank IRC
- discussions create a resource for future use that is easier to access than IRC logs
- IMO, this is a better outlet for the kinds of questions we get

Reply:
Copy that!